charts/generic-govuk-app/templates/_helpers.tpl

```diff
@@ -48,6 +48,7 @@ Selector labels
 */}}
 {{- define "generic-govuk-app.selectorLabels" -}}
 app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/repoName: {{ .repoName | default .Release.Name  }}
```

Comment:
This belongs in the normal labels, not the selector ones

Reply:
ok I'll move it up